tutorials/[English] Nebulas 101 - 01 Installation.md

```diff
@@ -10,11 +10,11 @@ To learn about Nebulas, please read the Nebulas [Non-Technical White Paper](http
 
 To learn about the technology, please read the Nebulas [Technical White Paper](https://nebulas.io/docs/NebulasTechnicalWhitepaper.pdf) and the Nebulas [github code](https://github.com/nebulasio/go-nebulas).
 
-> Nebulas can only runs on Mac and Linux at this stage. The Windows version will be coming soon.
+> Nebulas can only run on Mac and Linux at this stage. The Windows version is be coming soon.
```

Comment:
“is be coming soon” is not right.

Reply:
Oops, failed to delete the whole string. My bad!